Dockerfile

```diff
@@ -19,4 +19,4 @@ ENTRYPOINT ["entrypoint.sh"]
 EXPOSE 3000
 
 # Start the main process.
-CMD ["rails", "server", "-b", "0.0.0.0"]
+CMD ["rails", "server", "-b", "0.0.0.0", "-e", "development"]
```

Comment:
what the -e flags does?

Reply:
![image](https://user-images.githubusercontent.com/61894952/191874679-56e1b3c7-d10f-471d-aecf-9958da09488a.png)